vent/api/actions.py

```diff
@@ -341,6 +341,9 @@ def start(self, tool_dict):
                                                      str(container.short_id))
                                 except Exception as err:  # pragma: no cover
                                     self.logger.error(str(err))
+                                    # add restart=always for core containers
+                                    if tool_dict[container_tuple[1]]['labels']['vent.groups'].find('core') != -1 and container_tuple[1].find('rmq-es-connector') == -1:
```

Comment:
there is more than one place that `containers.run` can happen, so this probably isn't the best place for this check since it would duplicate the code to check in more than one place.  I would look inside the `prep_start` function and see where other dict keys are added when `groups` exist, and then check if `core` is in groups and then apply the `restart_policy`.

also, not sure why the specific check for `rmq-es-connector`?